unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarLOD/AvatarsLODController.cs

```diff
@@ -164,6 +184,14 @@ private float SqrtDistanceToOwnPlayer(Player player, Vector3 ownPlayerPosition)
 
         private bool IsInFrontOfCamera(Vector3 position) { return Vector3.Dot(cameraForward, (position - cameraPosition).normalized) >= RENDERED_DOT_PRODUCT_ANGLE; }
 
+        private void LoadTestPanel()
+        {
+            if (GameObject.Find("AvatarImpostorsTestPanel") != null)
+                return;
+
+            GameObject.Instantiate(Resources.Load<GameObject>("Prefabs/AvatarImpostorsTestPanel"));
+        }
+
```

Comment:
I would get rid of this for production.

Reply:
Removed Test panel, moved it into [another branch](https://github.com/decentraland/unity-renderer/pull/1111)